discord/ui/select.py

```diff
@@ -418,6 +485,8 @@ class UserSelect(BaseSelect[V]):
         ordering. The row number must be between 0 and 4 (i.e. zero indexed).
     """
 
+    __item_repr_attributes__ = BaseSelect.__item_repr_attributes__ + ('default_values',)
```

Comment:
I'm not big on this being in the `__repr__`. I mentioned this in another PR but I know this was done because for some reason the factory uses this as a way to know how to construct it. This should change, but it's outside of the scope of the PR.